ui/web/index.html

```diff
@@ -3,10 +3,307 @@
+                        <option value="A">A</option>
+                        <option value="B">B</option>
+                        <option value="C">C</option>
+                        <option value="D">D</option>
+                        <option value="E">E</option>
+                        <option value="F">F</option>
+                        <option value="G">G</option>
+                        <option value="H">H</option>
+                        <option value="I">I</option>
+                        <option value="J">J</option>
+                        <option value="K">K</option>
+                        <option value="L">L</option>            
+                    </select>
+                </td>
+                </td>
+            </tr>
+        </table>
+    </div> 
+
+    </fieldset>
+
+    <fieldset>
+
+        <legend>Load</legend>
+
+        <div id="place">
+
+            <input type="submit" name="but" id="loadTasksButton" value="Load">
+
+        </div>
+
+    </fieldset>
+    <fieldset>
+
+        <legend>Print</legend>
+
+            <table border="0" widtd="100%">
+                <tr>
+                    <td>Tasks
+                        <div id="choiceGraphic"></div>                      
+                    </td>
+                    <td>
+                        <input type="button" value="Print" id="printGraphicButton">
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <div id="chartdiv" style="height:500px;width:300px; "></div>
+                    </td>
+                </tr>
+                </table>
+    </fieldset>
+
+    <script> 
+            var applicationContext = {}; 
+                                              
+        function loadTasks() {
+            var excelFile = $('#excelFile').get(0).files.item(0);
+            $.ajax({
+                type: 'PUT',
+                url: 'http://localhost:5000/api/excelWorkbook',
+                data: excelFile,
+                contentType: 'application/vnd.ms-excel',
+                processData: false,
+                success: function (data) {
+                    var requestBody = {
+                        sheet: $("#sheetName").val(),
+                        first_row: +$("#firstRow").val(),
+                        last_row: +$("#lastRow").val(),
+                        rows_to_skip: $("#rowSkip").val(),
+                        columns_mapping: {
+                            uid: $("#uid").val(),
+                            name: $("#nam").val(),
+                            blockers: $("#blockers").val(),
+                            min_estimate: $("#minEst").val(),
+                            normal_estimate: $("#norEst").val(),
+                            max_estimate: $("#maxEst").val()
+                        }
+                    };
+                    alert("Response received: " + JSON.stringify(data));
+                    requestBody.file = data.tempFileName;
```

Comment:
It's always better to thinks in singe manner, so please move this line above to `requestBody` declaration:

```
var requestBody = {
                        file: data.tempFileName,
                        sheet: $("#sheetName").val(),
                        first_row: +$("#firstRow").val(),
                        last_row: +$("#lastRow").val(),
                        rows_to_skip: $("#rowSkip").val(),
                        columns_mapping: {
                            uid: $("#uid").val(),
                            name: $("#nam").val(),
                            blockers: $("#blockers").val(),
                            min_estimate: $("#minEst").val(),
                            normal_estimate: $("#norEst").val(),
                            max_estimate: $("#maxEst").val()
                        }
                    };
```

Reply:
Line moved to requestBody declaration